tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/jaypipes/ghw/pkg/snapshot"
+)
+
+func main() {
+	debug := flag.Bool("debug", false, "enable debug output")
+	output := flag.String("output", "", "path to clone system information into")
+	flag.Parse()
+
+	if *output == "" {
+		log.Fatal("--output is required")
+	}
+
+	if *debug {
+		snapshot.SetTraceFunction(func(msg string, args ...interface{}) {
+			log.Printf(msg, args...)
+		})
+	}
+
+	scratchDir, err := ioutil.TempDir("", "perf-must-gather-*")
+	if err != nil {
+		log.Fatalf("error creating temporary directory: %v", err)
```

Comment:
then we need to bail out?

Reply:
yes we do: https://golang.org/src/log/log.go?s=7078:7134#L196